odxtools/cli/_print_utils.py

```diff
@@ -19,14 +39,13 @@ def format_desc(desc: str, ident: int = 0) -> str:
     return desc
 
 
-def print_diagnostic_service(
-    service: DiagService,
-    print_params: bool = False,
-    print_pre_condition_states: bool = False,
-    print_state_transitions: bool = False,
-    print_audiences: bool = False,
-    allow_unknown_bit_lengths: bool = False,
-) -> None:
+def print_diagnostic_service(service: DiagService,
```

Comment:
original code style of keeping first argument in its own lime is what I personally prefer, but the final say on such topics go to @andlaus

Reply:
I agree, but I think this was imposed by yapf or else the workflow would complain. (the point of enforcing yapf's style is to avoid loosing to much time with discussions about coding style.) That said, maybe we should have a hard look at yapf's options in the not too distant future...

Reply:
@kakoeh did you change the argument position to be on the same line as the function, or did yapf did that on its own?

Reply:
I didn't change it, this was performed by yapf